katas/content/nonlocal_games/ghz_quantum_strategy/index.md

```diff
@@ -0,0 +1,8 @@
+**Inputs:**
```

Comment:
Since this task has three separate operations to implement, I'd suggest phrasing it the way we did in Classical Strategy exercise, clarifying that the learner needs to implement all three and that the inputs to them will be different

Reply:
Added sentence:
In this task, you should implement three functions, one for each player's quantum strategy.
Note that they are covered by one test, so you must implement all of them to pass the test.